main.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'app'
+
+def main
+  app = App.new
+
+  puts 'Welcome to School Library App!'
+
+  options = {
+    'List all books' => :list_books,
+    'List all people' => :list_people,
+    'Create a person' => :create_person,
```

Comment:
[OPTIONAL]
- [ ] These two were reversed,  when hitting the `2` instead of listing the person, it prompts the user to create a person. Kindly make sure you change that for a better user experience. :+1:

Reply:
-  [ ] Create person and create book also were reversed.

Reply:
- [ ] `KINDLY CONSIDER REVISING ALL YOUR OPTIONS FOR A BETTER USER EXPERIENCE, they are messy` and 
- [ ] Also consider adding the missing ones like `listing the person` and `the books`